gradlew

```diff
@@ -130,10 +130,13 @@ location of your Java installation."
     fi
 else
     JAVACMD=java
-    which java >/dev/null 2>&1 || die "ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
+    if ! command -v java >/dev/null 2>&1
```

Comment:
Why was this changed?

Reply:
this change was done by running the gradle wrapper

Reply:
👍